### ISSUE TITLE
Feature: save and upload

### ISSUE THREAD

**Issue by UziTech** [icetee/remote-ftp#399]
I don't like to automatically upload on every save but it would be nice to have a `remote-ftp:save-and-upload` command once the file is ready to be uploaded

**Comment by amc123-glitch** [comment]
I have to agree with this. The need for an upload on save would help since if you disconnect it will still attempt to connect to the server the first few times you hit save.

**Comment by jimmaaay** [comment]
In the package settings you can set `Auto Upload On Save` to either `never` or `only when connected`. Then when the file is saved you can run the command `Remote Ftp: upload selected`

**Comment by UziTech** [comment]
I believe `Remote Ftp: upload selected` only uploads files that are selected in the tree view. The `remote-ftp:save-and-upload` should upload the active file in the editor

**Comment by jimmaaay** [comment]
If you have a file open it will be selected in the tree view unless you specifically select something else

**Comment by UziTech** [comment]
Actually it will only automatically select active files if Auto Reveal is checked in the tree view setting which is defaulted to false.
![image](https://cloud.githubusercontent.com/assets/97994/19238290/7dc3a5e0-8ec6-11e6-9f76-e91f8d1a8f5a.png)

**Comment by jimmaaay** [comment]
It's false for me and it still  works

**Comment by jimmaaay** [comment]
I see. If a file is in a directory that is closed then it will select the folder,  but if it's open it will select the file
